keyboards/dz60/keymaps/LEdiodes/keymap.c

```diff
@@ -1,108 +1,208 @@
+/*
+	Create By:LEdiodes
+	Created On:05-07-2018 2:55 PM CST
+
+		Wants;
+			Live recording(dynamic Macros)
+			Common Ticket Notes added
+			more double and triple taps to complete thing
+
+Keymap: 32 Layers
+-----------------
+stack of layers
+       ____________ precedence
+      /           / | high
+  31 /___________// |
+  30 /___________// |
+  29 /___________/  |
+   :   _:_:_:_:_:__ |
+   :  / : : : : : / |
+   2 /___________// |
+   1 /___________// |
+   0 /___________/  V low
+*/
+
 #include "dz60.h"
```

Comment:
Please replace all of the includes with `#include QMK_KEYBOARD_H`, as this will cover everything.